From 0458dc32aa669f2c2a8b4b1402e05a447579d339 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Sat, 24 Nov 2018 18:38:35 +0800
Subject: [PATCH 2/9] nxp-ls146a:remove no-map attribute for reserved memory

When performing kexec, qman/bman driver need access reserved memory to
determine device state in second kernel. So remove no-map attributes to
allow kernel access those area.

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
index 290aaf5..6f321b0 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
@@ -772,19 +772,16 @@
 			compatible = "shared-dma-pool";
 			size = <0 0x1000000>;
 			alignment = <0 0x1000000>;
-			no-map;
 		};
 		qman_fqd: qman-fqd {
 			compatible = "shared-dma-pool";
 			size = <0 0x800000>;
 			alignment = <0 0x800000>;
-			no-map;
 		};
 		qman_pfdr: qman-pfdr {
 			compatible = "shared-dma-pool";
 			size = <0 0x2000000>;
 			alignment = <0 0x2000000>;
-			no-map;
 		};
 	};
 
-- 
1.7.9.5

